releases/net471/KnownIssues/510901-Apps-targeting-.NET-4.6.1-who-using-.NET-Standard-libraries-might-be-broken-on-.NET-4.7.1.md

```diff
@@ -0,0 +1,16 @@
+## Symptoms
+
+Applications that target the .NET Framework 4.6.1 and use .NET Standard libraries may experience <xref:TypeLoadExceptions> exceptions when running on .NET 4.7.1
+
+## Cause
+
+The .NET Framework 4.7.1 adds support for .NET Standard in-box. As part of this change, a few assemblies that used to ship independently from the .NET Framework have now been added to the .NET Framework. As these .NET Standard assemblies were not part of the .NET Framework prior to .NET 4.7.1 they were deployed with the application, and a binding redirect was added to the application's app.config.
+
+When running on the .NET Framework 4.7.1, where the assemblies are now part of the Framework, the application now has two types with different identities and, that leads to a <xref:TypeLoadException>. One type comes from the app-local assembly and the other one comes from the .NET Framework assembly.
```

Comment:
the comma should be before "and", not after
there also should be a comma between "app-local assembly" and "and the other"